packages/pds-ember/README.md

```diff
@@ -0,0 +1,33 @@
+# pds-ember
+
+Ember addon for building Structure-styled UIs.
+
+
+# Compatibility
+
+* Ember.js v3.16 or above
+* Ember CLI v2.13 or above
+* Node.js v10 or above
+
+
+# Installation
+
+```
+ember install pds-ember
```

Comment:
This snippet may need updated to reflect the NPM package namespace... `@hashicorp/pds-ember`.